### ISSUE TITLE
VLC/syncplay closes if a user joins and a video is currently in progress

### ISSUE THREAD

**Issue by mechchicken** [Syncplay/syncplay#213]
The user joining a room will crash if there is a video in a room currently being played. 

Some details on this:
- 1.6.1 crashes, but works fine on version 1.5.2 (these are the only two that I tested)
- Only happens with VLC. Other media players are fine.
- Does NOT crash if the the path to the video is entered in the "Media Player Settings" under "Path to video (Optional)"
- No crash logs are created in either syncplay or VLC.

**Comment by Et0h** [comment]
Thanks for reporting this issue, hopefully we can get to the bottom of it.

I tried to reproduce using VLC 3.0.4 on Windows 10 by adding the file downloaded from BigBuckBunny_320x180.mp4  available from https://download.blender.org/peach/bigbuckbunny_movies/ to the playlist on one instance of Syncplay and joining the room with another instance, but it did not crash. I tried both when the video was stopped and it was playing. The version of Syncplay.lua in C:\Program Files\VideoLAN\vlc\lua\intf\syncplay.lua is 0.3.4

Please:
- Try with a specified file that is available online, e.g. a specific version of https://peach.blender.org/download/ that I can also test.
- Try with MPC-HC or VLC to confirm the issue only affects VLC. If it only affects VLC, please specify the version of VLC.
- Check %APPDATA%\.syncplay.log for any crash log.
- State what Operating System you are using.
- Provide more detailed guide for reproduction.

**Comment by Et0h** [comment]
As we have had only one report of this issue, we have not been able to reproduce the issue, and do not have the requisite information to investigate it further (see earlier post) I am closing this issue for now.

However, if anyone does experience this crashing issue with Syncplay 1.6.2+ then please let me know, taking account the five bullet points I set out above.

**Comment by albertosottile** [comment]
Reproduced (Syncplay 1.6.1 - macOS 10.13.6 - VLC 3.0.4)
```
error: uncaptured python exception, closing channel <__Listener(VLC Listener, started daemon 123145421053952)> (<class 'TypeError'>:'>' not supported between instances of 'NoneType' and 'int' [asyncore.pyc|read|83] [asyncore.pyc|handle_read_event|423] [asynchat.pyc|handle_read|171] [syncplay/players/vlc.pyc|found_terminator|488] [syncplay/players/vlc.pyc|lineReceived|232])
```
The funny thing is that happens only the first time I install a brand new copy of Syncplay. If I try to reproduce it again, the bug does not trigger.

**Comment by albertosottile** [comment]
Fixed by https://github.com/Syncplay/syncplay/commit/e1af902f7c7e2a4536f41c86c99bd0c578a70178 (fix actually tested on macOS 10.13.6).
